### ISSUE TITLE
Scala Steward Action on a single project fails

### ISSUE THREAD

**Issue by bpg** [scala-steward-org/scala-steward-action#21]
I'm trying to setup V2 of the action on my (single) project, but have no luck.
```
Run scala-steward-org/scala-steward-action@v2
✓ Connected to Maven Central
✓ Coursier installed, version: 2.0.0-RC6-17
✓ Github Token provided as input
##[error] ✕ The path indicated in `repos-file` () does not exist
```
A simple MVP that produces the same error:
https://github.com/bpg/ss-gh-test/actions/runs/108809187

**Comment by bpg** [comment]
I'm not a JS expert by any means, but it could be a regression after https://github.com/scala-steward-org/scala-steward-action/pull/15/

**Comment by bpg** [comment]
Adding `repos-file` or `github-repository` parameters does not help, action fails with the same error.
V1 of the action seems to be working.

**Comment by alejandrohdezma** [comment]
Hi @bpg thank you very much for this 😸 

This problem was indeed a bug introduced in #15 in the way inputs were being handled.

I sent a fix for it in #25.

I'll merge and publish a new `v2` version once tests passed.

Thanks!

**Comment by bpg** [comment]
Thanks for the prompt response! 👍
